tests/modules/nf-core/custom/tabulartogseagct/main.nf

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { CUSTOM_TABULARTOGSEAGCT } from '../../../../../modules/nf-core/custom/tabulartogseagct/main.nf'
+
+process tsv_to_csv {
+
+    input:
+    path expression_matrix
+
+    output:
+    path 'test.csv'
+
+    script:
+    """
+    sed 's/\t/,/g' ${expression_matrix} > test.csv.tmp
+    mv test.csv.tmp test.csv
+    """
+
+}
+
+// Example expression matrix comes from rnaseq workflow, which puts symbols in
+// the second column, strip columns
+
+process strip_column {
+
+    input:
+    path tsv
+
+    output:
+    path 'test.tsv'
+
+    script:
+    """
+    cut -f2 --complement $tsv > test.tsv
+    """
+}
+
+workflow test_custom_tabulartogseagct {
+    
+    input = strip_column(file(params.test_data['mus_musculus']['genome']['rnaseq_matrix'], checkIfExists: true))
+        .map{
+            tuple([ id:'test' ], it)
+        }
+
+    CUSTOM_TABULARTOGSEAGCT ( input )
+}
```

Comment:
These could be simplified using Groovy code: https://www.nextflow.io/docs/latest/script.html in combination with CollectFile (https://www.nextflow.io/docs/latest/operator.html?highlight=collectfile#collectfile), this way these files won't be in the output directory (and the tests won't pick them up). This would also be better for reproducibility as these may not work on all systems